cfme/tests/services/test_provision_stack.py

```diff
@@ -26,73 +26,127 @@
 AWS_TEMPLATE = """
```

Comment:
There is a data directory and a file loader to help with this, but in a second PR is fine